packages/appstore/src/modules/onboarding/onboarding.scss

```diff
@@ -109,9 +109,12 @@
     &__header {
         width: 100%;
         height: 7.2rem;
-        background-color: $color-black;
         display: flex;
         justify-content: center;
         align-items: center;
+
+        &--mobile {
+            height: auto;
+        }
```

Comment:
```suggestion
        @include mobile {
            height: auto;
        }
```

Reply:
@heorhi-deriv Great! Thank you for this, it was really stupid from my side